ase/io/formats.py

```diff
@@ -351,6 +351,10 @@ def get_ioformat(name: str) -> IOFormat:
   module='espresso', ext=['pw2wan', 'p2wi'])
 F('pw2wannier-out', 'pw2wannier output file', '1F',
   module='espresso', ext='p2wo')
+F('wann2kcp-in', 'wann2kcp input file', '1F',
+  module='espresso', ext=['w2kcp', 'wki'])
```

Comment:
is there any reason why you've kept `w2kcp` as a valid ext?

Reply:
Sorry, my bad!